torch/ao/ns/fx/mappings.py

```diff
@@ -195,6 +195,10 @@ def get_base_name_to_sets_of_related_ops() -> Dict[str, Set[NSNodeTargetType]]:
         set([
             F.hardswish,
         ]),
+        # F.group_norm
```

Comment:
I added this to get a test to pass, but I don't know if I need to do anything else other than modify the dict

Reply:
not sure why we need this, cc @vkuzo to check